doc/nrf/gs_installing.rst

```diff
@@ -219,6 +219,51 @@ Therefore, remember to regularly check for updates:
          pip3 install -U west
 
 
+Installing additional toolchains for Connected Home over IP applications development
+====================================================================================
+
+.. note::
+   If you are not interested in the `Connected Home over IP`_ applications, you can ommit steps presented in this section.
+
+`Connected Home over IP`_ project is using `GN`_ meta-build system to generate files for `Ninja`_ and both tools are necessary in order to build an applications.
```

Comment:
did you mean "in order to build a Connected Home over IP application"?

Reply:
Yes, isn't it clear enough while whole section applies to the Connected Home over IP?

Reply:
I'm not sure. Probably @greg-fer will have comments on that ;). But please fix the grammar error: "an applications" -> "an application".